### PR TITLE
Multiple minor clean-ups from previous PR.

### PR DESCRIPTION
**Pull request by mirelap-amazon**
*Issue #, if available:*

*Description of changes:*
- Responded to comments from previous [PR #2](https://github.com/aws/amazon-codeguru-profiler-python-agent/pull/2).
- Added links to PyPI repository and demo application

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 For more details, check the documentation: https://docs.aws.amazon.com/codeguru/latest/profiler-ug/what-is-codeguru-profiler.html
 
+## How to use it
+
+This package is released to PyPI, so use it as any Python package from PyPI: https://pypi.org/project/codeguru-profiler-agent
+
+For a demo application that uses this agent, check our [demo application](https://github.com/aws-samples/aws-codeguru-profiler-demo-application).
```
Comment (gimki) [inline README.md:9]: I think we should link to: https://github.com/aws-samples/aws-codeguru-profiler-python-demo-application

File: test/integration/test_live_backend_reporting.py [hunk 2]
```diff
@@ -18,8 +18,6 @@
 
 
 @pytest.mark.skipif(
-    # TODO FIXME Remove the conditions for skipping this on Amazonian fleets when we move fully to GitHub.
-    socket.gethostname().startswith("pb-worker-prod") or
     socket.gethostname().startswith("coverlay-") or
```
Comment (gimki) [inline test/integration/test_live_backend_reporting.py:21]: Should we remove this as well(?) This is something internal right?

File: README.md (hunk 1 above)
Comment (mirelap-amazon) [reply README.md]: Good catch, I will update.

File: test/integration/test_live_backend_reporting.py (hunk 2 above)
Comment (mirelap-amazon) [reply test/integration/test_live_backend_reporting.py]: Yes, I will remove it.

**Pull request merged**